packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1145,20 +1158,28 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   @override
   Widget build(BuildContext context) {
     updateScrollbarPainter();
+
+    Widget child = CustomPaint(
+      key: _scrollbarPainterKey,
+      foregroundPainter: scrollbarPainter,
+      child: RepaintBoundary(child: widget.child),
+    );
+
+    if (!_isMobile) {
+      // Hover events not supported on mobile.
+      child = MouseRegion(
+        onExit: handleHoverExit,
+        onHover: handleHover,
```

Comment:
We can use `PointerEvent.kind` to distinguish the type of input device to deal with this issue. Moreover, the mobile phone can also be used with an external mouse. At this time, we can better respond to the hover event.